nuget/deprecated/nunit.engine.netstandard.nuspec

```diff
@@ -9,7 +9,7 @@
     <license type="file">LICENSE.txt</license>
     <projectUrl>https://nunit.org</projectUrl>
     <repository type="git" url="https://github.com/nunit/nunit-console"/>
-    <iconUrl>https://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_256.png</iconUrl>
+    <icon>images\nunit_256.png</icon>
```

Comment:
Not much point here, since I assume we don't issue new versions of the deprecated package.

Reply:
Yes, I guess we can skip here. I had modified it just to be consistent.

Reply:
Personally, I would go for consistency.

Reply:
But if you don't publish it isn't that an even worse inconsistency?

Reply:
Am ok with either way. But I think @CharliePoole  has a point. Let me revert the change for the deprecated package.

Reply:
@mano-si To be clear, I was __asking__ whether the project releases new versions of the deprecated package. I don't know the answer but just assumed they don't. It's a manual process, BTW.

Reply:
The package was also published in the latest release - version 3.11.1- https://www.nuget.org/packages/nunit.engine.netstandard/.

Reply:
I see you're maintaining the packages even though they are deprecated. That's not what I assumed, and it's not how I thought deprecation was supposed to work. If you update those packages every release, then exactly what does it mean that they are deprecated? Rather than deprecated, it seems like they are just aliases for the newer package names. That's certainly an approach you can take, but I wouldn't call it deprecation.

In any case, since my assumption was wrong, I'll back out of this discussion. Now that GitHub has an official notion of deprecating packages, you might want to look at that at some point in the future.

Reply:
Deprecated packages are still being released for now as a low-cost solution to keeping people on the latest version. 🙂 #720 tracks removing them properly.

Thanks for updating these as well, @mano-si!